<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamSalleDial</class>
 <widget class="QDialog" name="ParamSalleDial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètres</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/res/param.png</normaloff>:/res/param.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #737373;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image :none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/res/engrenage_big.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>35</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image :none;</string>
       </property>
       <property name="text">
        <string>Paramètres</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-image :none;</string>
     </property>
     <property name="title">
      <string>Salle et murs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelParamSalle">
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <property name="text">
           <string>Hauteur des murs</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinParamSalle">
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>30.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QWidget" name="widgetTexture" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="labelParcourir">
               <property name="text">
                <string>emplacement de l'image...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="boutonParcourir">
               <property name="text">
                <string>Parcourir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Hauteur de l'image réelle : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinSize">
               <property name="suffix">
                <string> m</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <property name="text">
           <string>Texture du sol</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="listeSol">
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <item>
           <property name="text">
            <string>Aucun</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Moquette</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Parquet</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Importer une texture</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Activer l'apercu des tableaux affichés au survol du mur</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">background-image :none;</string>
     </property>
     <property name="title">
      <string>Moteur 3D</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <property name="text">
           <string>Reconfigurer les paramètres de rendu</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="boutonReconfigurer">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <property name="text">
           <string>Reconfigurer</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/res/clear.png</normaloff>:/res/clear.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">background-image :none;</string>
     </property>
     <property name="title">
      <string>Projet</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <property name="text">
           <string>Reconstruire les miniatures des images</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="boutonReconstruire">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image :none;</string>
          </property>
          <property name="text">
           <string>Reconstruire</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/res/passerVueFace.png</normaloff>:/res/passerVueFace.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progression">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image :none;</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%p% effectué</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="styleSheet">
      <string notr="true">background-image :none;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>535</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">background-image :none;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ParamSalleDial</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParamSalleDial</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
